AdminWebsite/AdminWebsite/ClientApp/src/app/testing/mocks/MockOidcSecurityService.ts

```diff
@@ -0,0 +1,43 @@
+import { Observable, of } from 'rxjs';
+interface UserData {
+    preferred_username?: string;
+    name?: string;
+    email?: string;
+    email_verified?: false;
+    given_name?: string;
+    role?: string;
+    amr?: string;
+}
+
+export class MockOidcSecurityService {
+    userData: UserData;
+    authenticated: boolean;
```

Comment:
if adding additional tests will need to add configuration in here too, can copy from video web

Reply:
added